src/dash.ts

```diff
@@ -30,12 +29,19 @@ export class Dash {
       .map(docset => `${this.option.exactDocset ? 'exact:' : ''}${docset}`)
       .join(',');
     const encodedQuery = encodeURIComponent(query);
-    return `${this.URIHandler} "dash-plugin://query=${encodedQuery}${keys
-      ? `&keys=${keys}`
-      : ``}"`;
+    return `${this.URIHandler} "dash-plugin://query=${encodedQuery}${
+      keys ? `&keys=${keys}` : ``
+    }"`;
   }
 }
 
 export interface DashOption {
   exactDocset: boolean;
 }
+
+interface OSOptions {
+  darwin: string;
+  linux: string;
+  win32: string;
+  [key: string]: string;
```

Comment:
I believe we can remove this since no other OS supported beside those three.

Reply:
👍 i'll try another approach